about.html

```diff
@@ -0,0 +1,162 @@
+      <section class="intro-about">
+        <div class="intro-title-about">
+          <h1>CONVECTION ABOUT CRIPTOS AND THE NEW ORDER</h1>
+        </div>
+        <div class="intro-text-about">
+          <p>
+            Our first big conference in Venezuela to talk about cryptos with the
+            best of the best! Thanks to our partner agencies around the globe,
+            we are able to make the most big conference in latin America. They
+            say they critpcurrencies are the future. They assure us they want to
+            help us become a better civilization. They teach us about the new
+            tecnologies to come. Are you ready? Sign up and prepare to the
+            future!
+          </p>
+        </div>
+        <div class="intro-contact">
+          <p>
+            Please contact us per Email for any further questions about our
+            Criptocurrencie conference!
+          </p>
+          <p class="contact-email"><strong>criptosfuture@gmail.com</strong></p>
+        </div>
+      </section>
+
+      <section class="logo-about">
+        <div class="title-logo">
+          <h2>Crypto and the New Order logo 2022</h2>
+          <div class="orange-line"></div>
+          <p>
+            The logo for the convection "CRYPTO AND THE NEW ORDER" was decided
+            from a group of the finest dessigners
+          </p>
+        </div>
+        <div class="logo-container">
+          <img src="images/logo.png" alt="Logo crypto and the new order" />
+        </div>
+      </section>
+
+      <section class="past-convention">
+        <div class="past-title">
+          <h2>See the past Crypto and New Order conventions</h2>
+          <div class="orange-line"></div>
+          <p>
+            Take a look to the last two Crypto and New Order wich took place in
+            Spain and Argentina
+          </p>
+        </div>
+        <div class="past-convention-card">
+          <div class="spain card-past">
+            <h3>2010</h3>
+            <h4>Crypto and New Order Barcelona Spain</h4>
+          </div>
+          <div class="argentina card-past">
+            <h3>2018</h3>
+            <h4>Crypto and New Order Buenos Aires Argentina</h4>
+          </div>
+        </div>
+      </section>
+
+      <section class="partner-section">
+        <div class="partner-title">
+          <h2>Partner</h2>
+          <div class="orange-line"></div>
+        </div>
+        <div class="partners-logo-container">
+          <ul class="partners-logo">
+            <li><img src="images/binance-logo.png" alt="binance" id="binance"></li>
+            <li><img src="images/crypto-com.png" alt="crypto.com" id="crypto"></li>
+            <li><img src="images/coingecko-logo.png" alt="coingecko" id="coingecko"></li>
+            <li><img src="images/coinmarketcap-logo.png" alt="coinmarketcap" id="coinmarketcap"></li>
+            <li><img src="images/coinbase-logo.png" alt="coinbase" id="coinbase"></li>
+          </ul>
```

Comment:
Kindly fix the partner's section image size in the `mobile version` as the previous reviewer asked to match the Behance design.

| Your design | Behance design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/24703171/163515695-a0a757c2-53ba-4ac7-b672-1265e8adc444.png) | ![image](https://user-images.githubusercontent.com/24703171/163515727-9f614189-1699-4a34-99cc-932d15377f1a.png) |

You can do that by giving the li that contains the image a specific width with percentages like:
```
.partners-logo li {
  width: 30%;
}
```
Remember to override this style in the desktop version to not corrupt it as it now looks perfect.